stargazer/APIReference/constellationRPCAPI/wallet_watchAsset.md

```diff
@@ -0,0 +1,66 @@
+---
+hide_table_of_contents: true
+---
+
+<head>
+  <meta
+    name="description"
+    content="Adds an L0 token to the Stargazer wallet."
+  />
+</head>
+
+<intro-end />
+
+Adds an L0 token to the Stargazer wallet.
+
+##### Parameters
+
+| Name | Type                           | Description               |
+| ---- | ------------------------------ | ------------------------- |
+| Data | `Object<WatchAssetParameters>` | The L0 token info object. |
+
+```typescript title="WatchAssetParameters"
+type WatchAssetParameters = {
+  type: String; // The token's interface. "L0" is the only supported type.
+  options: WatchAssetOptions;
+};
+```
+
+```typescript title="WatchAssetOptions"
+type WatchAssetOptions = {
+  chainId: Number; // The chain ID. 1 (mainnet), 3 (testnet), 4 (integrationnet)
+  address: Address; // Metagraph address
+  l0: String; // L0 endpoint
+  l1: String; // L1 endpoint
+  name: String; // Name of the token
+  symbol: String; // Symbol of the token
+  logo: String; // Logo of the token
+};
+```
+
+##### Return Type
+
+`String<Boolean>` - True if the token was added successfully.
+
+##### Example
+
+```typescript title="TypeScript"
+await provider.request({
+  method: "wallet_watchAsset",
+  params: [
+    {
+      type: "L0",
```

Comment:
What are the other "types" used for EVM networks? I'm not sure "L0" is the best option here.

Reply:
[EIP-747](https://eips.ethereum.org/EIPS/eip-747) defines two types for EVM networks: `ERC20` and `ERC1046`. We can definitely change the `L0` type used here.